### ISSUE TITLE
Missing data (fits images) for S5.2

### ISSUE THREAD

**Issue by timstaley** [griffinfoster/fundamentals_of_interferometry#88]
The 'data/fits' folder is used a lot for the 'Sampling and Point Spread Functions' section, but has not been committed.

A quick suggestion - if you're going to add a significant amount of binary-blob data, you might want to add it as a git submodule, so that checking out the original git repository doesn't require an initial large download. On the other hand, submodules add complexity and help to confuse newcomers, so, pros and cons.

**Comment by griffinfoster** [comment]
Yeah, this is a problem in which I haven't decided on a solution to. Currently, there is a link to the files (http://www.mth.uct.ac.za/~siphelo/admin/interferometry/data/fundamentals_fits.tar.gz) and some notes in the readme on where to include the files.

A git submodule is a good idea I hadn't thought of. There is also the github lfs extension.

**Comment by timstaley** [comment]
Ah, I should have RTFM (aka. Read The Fine Readme). Ta :-)

**Comment by lao19881213** [comment]
This http://www.mth.uct.ac.za/~siphelo/admin/interferometry/data/fundamentals_fits.tar.gz not wokrs:

_wget http://www.mth.uct.ac.za/~siphelo/admin/interferometry/data/fundamentals_fits.tar.gz
--2019-11-06 23:31:38--  http://www.mth.uct.ac.za/~siphelo/admin/interferometry/data/fundamentals_fits.tar.gz
Resolving www.mth.uct.ac.za (www.mth.uct.ac.za)... 137.158.154.230
Connecting to www.mth.uct.ac.za (www.mth.uct.ac.za)|137.158.154.230|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2019-11-06 23:31:38 ERROR 404: Not Found._
